### DISCUSSION TITLE
Wazuh sysmon alerts stopped after upgrade to 2.3.120

### DISCUSSION THREAD

**Discussion by Phillip-Samson** [Security-Onion-Solutions/securityonion#7882] [Unsupported Versions] [answered]
Standalone
2.3.120

After performing the 2.3.120 upgrade I noticed that Wazuh alerts are coming in still but, sysmon alerts are not showing up, anywhere. I have seen [Issue 7493](https://github.com/Security-Onion-Solutions/securityonion/discussions/7493#discussion-3928050) however compared to that issue I do not see any alert anywhere. 
![image](https://user-images.githubusercontent.com/36277967/166121573-4a1d8bd7-0fe7-42ef-a991-efa627723987.png)

**Comment by dougburks** [comment, marked as answer]
Have you verified that sysmon is still generating logs on the endpoint?

Have you checked the Wazuh agent logs on the endpoint?

Have you checked the logs in `/opt/so/log/` on the Security Onion box that your Wazuh agent connects to?

**Reply by Phillip-Samson** [reply]
I had checked all of those things and everything looked good. I had also checked the firewall and ran tcpdump to verify that everything was coming in, and it was so.....

I am not sure what happened but everything started working while I was OoO on Sunday. I ran a query on Monday morning and it all looked good. Thanks for your reply @dougburks
